config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -57,6 +57,20 @@ presets:
       items:
       - key: bootstrapper-key
         path: bootstrapper-key
+- labels:
+    preset-cluster-api-provider-vsphere-gcs-creds: "true"
+  volumes:
+  - name: cluster-api-provider-vsphere-gcs-creds
```

Comment:
Hi @codenrhoden,

Is there any reason to have the volume name different from that of the secret? I don't feel strongly about this at all -- just curious.